lib/DebugInfo/Symbolize/BugsnagSymbolize.cpp

```diff
@@ -68,19 +72,23 @@ SymbolizeResults BugsnagSymbolize(const char* filePath, bool includeInline, int6
   std::vector<SymbolizeResult> results;
 
   for (int i = 0; i < addressCount; i++) {
-    auto ResOrErr = Symbolizer.symbolizeCode(moduleName, addresses[i]);
+    std::string stringAddress = addresses[i];
+    char* addressErr;
+    int64_t numericAddress = (int64_t)std::strtoul(stringAddress.c_str(), &addressErr, 16);
+    auto ResOrErr = Symbolizer.symbolizeCode(moduleName, numericAddress);
+
     if (includeInline) {
-      auto ResOrErr = Symbolizer.symbolizeInlinedCode(moduleName, addresses[i]);
+      auto ResOrErr = Symbolizer.symbolizeInlinedCode(moduleName, numericAddress);
       if (ResOrErr) {
         for (int j = 0; j < ResOrErr.get().getNumberOfFrames(); j++) {
-          SymbolizeResult result = getSymbolizeResult(ResOrErr.get().getFrame(j), addresses[i], (j == 0) ? false: true);
+          SymbolizeResult result = getSymbolizeResult(ResOrErr.get().getFrame(j), stringAddress, (j == 0) ? false: true, (*addressErr == 0) ? false: true);
```

Comment:
Can `(*addressErr == 0) ? false: true)` be simplified to `*addressErr != 0`?

Or you could create a separate variable (something like `bool isBadAddress = *addressErr != 0` to make this logic clearer.

Same comment below